src/views/CohortPage.scss

```diff
@@ -0,0 +1,25 @@
+.cohort-page {
```

Comment:
I'm thinking that these classes and the other layout classes in HomePage.scss could be changed to styled components if we create wrapper styled components for use within the CohortPage and HomePage components.

So, for example, instead of having a div with className="sidebar", there could be a styled.div component called "SidebarContainer" that wraps `<Sidebar />` in CohortPage.js.

Reply:
I agree